<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="StudentsList">
        <property name="text">
         <string>Students List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DebtorsList">
        <property name="text">
         <string>Debtors List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DismissList">
        <property name="text">
         <string>Dismiss List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sort">
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLineEdit" name="mathW"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Winter mark:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Summer mark:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QTableWidget" name="debtors">
        <column>
         <property name="text">
          <string>debtors</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="histS"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="progS"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Prog:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="mathS"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="open">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>History:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QTableWidget" name="Dismiss">
        <column>
         <property name="text">
          <string>dismiss</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="newStud"/>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Math:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="histW"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="progW"/>
      </item>
      <item row="0" column="1">
       <widget class="QTableWidget" name="students">
        <column>
         <property name="text">
          <string>Group</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Students</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="stud"/>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="name">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="group">
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="winter">
        <property name="text">
         <string>Winter Session</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="summer">
        <property name="text">
         <string>Summer Session</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Output">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
